FirebaseCore/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# Firebase 10.0.0
+- [changed] **Breaking change**: Firebase's minimum supported versions have
```

Comment:
CocoaPods only for iOS and tvOS. For SPM,  only macOS only is updated to 10.13.  Binary distribution minimum versions are unchanged.

Reply:
Why not 10.13 for CocoaPods?

Reply:
> Binary distribution minimum versions are unchanged.

Does this mean I don't bump tvOS and macOS for binary pods?

Reply:
Zip already has the updates. See https://github.com/firebase/firebase-ios-sdk/blob/master/ReleaseTooling/Sources/ZipBuilder/main.swift#L103.

I mean macOS needs to be updated for SPM AND CocoaPods.
tvOS and iOS only need CocoaPods updates

Reply:
Done. PTAL.